flatgfa/src/cli/cmds.rs

```diff
@@ -299,3 +301,50 @@ pub fn gaf_lookup(gfa: &flatgfa::FlatGFA, args: GAFLookup) {
         }
     }
 }
+
+///
```

Comment:
Please add a short description of the command. This shows up in the `fgfa help` output, for example. You can see the other commands for a hint about how these are usually formatted (super short phrases).